docs/pages/documentation/11-examples/example-twitter-streaming-public-tweets.md

```diff
@@ -0,0 +1,580 @@
+# Using Grakn To Work With Tweet Data
+In this tutorial we will look at how to stream public tweets into Grakn's knowledge graph. The tutorial aims to demonstrate key concepts such as receiving, inserting and querying data. Upon the completion of this tutorial, you will have learnt about these concepts:
+
+- Defining a simple Grakn.ai ontology using the Java API
+- Streaming public tweets into the application with the [Twitter4J](http://twitter4j.org/ "Twitter4J") library
+- Inserting tweets into the knowledge graph using Grakn's Graph API
+- Performing simple queries using GraQL, the Grakn's Query Language
+
+## Registering Your Own Twitter Application
+As of today, you will need a valid credential in order to call practically every endpoint in the Twitter API. Therefore, you must own a Twitter application (or, register a new one) before proceeding further.
```

Comment:
How about `As of today, every endpoint in the Twitter API is authenticated. Therefore, you must own a Twitter application (or, register a new one) before proceeding further.`

3c70e4c5348342d4879374cff4cfd57bc8dd0264